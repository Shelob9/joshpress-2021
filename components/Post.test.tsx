import BlockContent from "@shelob9/block-content";
import { render } from '@testing-library/react';
import React from 'react';
describe("Post", () => {
    it ('Works with block content lib', () => {
        const { container } = render(<BlockContent rawContent={`<!-- wp:paragraph -->
        <p>First</p>
        <!-- /wp:paragraph -->`} />);
        expect(container).toMatchSnapshot();
    })

})