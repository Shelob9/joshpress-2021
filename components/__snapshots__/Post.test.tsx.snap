// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post matches snapshot 1`] = `
<div>
  <div>
    <p>
      First
    </p>
  </div>
</div>
`;
